recipes/graphvite/meta.yaml

```diff
@@ -0,0 +1,70 @@
+{% set build_number = "0" %}
+
+package:
+  name: graphvite
+  version: 0.1.0
+
+source:
+  url: https://github.com/DeepGraphLearning/graphvite/archive/master.zip
+  sha256: 3dd505d939204e41f762a18d51b3719e327a5fe461bce0dd0e8ce07cab37272b
+
+build:
+  skip: true  # [not linux]
+  number: {{ build_number }}
+  string: py{{ python|replace(".", "") }}cuda{{ cudatoolkit|replace(".", "") }}_{{ build_number }}
+
+requirements:
+  build:
+    # cmake
+    - cmake >=3.12
+    - {{ compiler("cxx") }}
+    - glog
+    - gflags
+    - cudatoolkit-dev {{ cudatoolkit }}
+    - python {{ python }}
+    - pybind11
+  host:
+    # make
+    - glog
+    - gflags
+    - cudatoolkit-dev {{ cudatoolkit }}
+    - python {{ python }}
+    - pybind11
+    - mkl >=2018
+    # setup
+    - numpy {{ numpy }}
+    - pyyaml
+    - easydict
+    - six
+  run:
+    - glog
+    - gflags
+    - cudatoolkit
```

Comment:
This is already added and properly constrained by `{{ compiler("cuda") }}` above.

```suggestion
```